tools/rna_tools/blockclust/blockclust.xml

```diff
@@ -1,88 +1,61 @@
-<tool id="blockclust" name="BlockClust" version="1.0.0">
+<tool id="blockclust" name="BlockClust" version="1.1.0">
     <description>efficient clustering and classification of non-coding RNAs from short read RNA-seq profiles</description>
     <requirements>
-        <requirement type="package" version="1.0">blockclust</requirement>
+        <requirement type="package" version="0.15.0">pysam</requirement>
         <requirement type="package" version="1.1">eden</requirement>
-        <requirement type="package" version="3.0.3">R</requirement>
-        <requirement type="package" version="0.1.19">samtools</requirement>
-        <requirement type="package" version="12.135">mcl</requirement>
-        <requirement type="package" version="1.0">blockclust_rlibs</requirement>
+        <requirement type="package" version="0.20.0">scikit-learn</requirement>
+        <requirement type="package" version="1.8.0">r-dendextend</requirement>
+        <requirement type="package" version="14.137">mcl</requirement>
+        <requirement type="package" version="1.1.0">blockclust</requirement>
     </requirements>
-    <version_command>echo '1.0'</version_command>
+    <version_command>blockclust.py -v</version_command>
     <command>
 <![CDATA[
+        export BLOCKCLUST_DATA_PATH=\$(dirname \$(which blockclust.py))/../share/blockclust_data;
         #if str($tool_mode.operation) == "pre":
-            BlockClustPipeLine.pl -m PRE -bam $tool_mode.reads_bam -tbed $tags_bed
-        #elif str($tool_mode.operation) == "clust":
+            blockclust.py -m PRE -bam $tool_mode.reads_bam -tbed $tags_bed;
```

Comment:
reads_bam and all other data-params as well as text-params and outputs should be single-quoted --> `' '`